src/sentry/monitors/models.py

```diff
@@ -312,6 +315,31 @@ class Meta:
     __repr__ = sane_repr("guid", "name")
 
 
+class MonitorEnvironmentManager(BaseManager):
+    """
+    A manager that consolidates logic for monitor enviroment updates
+    """
+
+    def ensure_environment(
+        self, project: Project, monitor: Monitor, environment_name: str | None
+    ) -> MonitorEnvironment:
+        if not environment_name:
```

Comment:
Should we allow empty string here? Or are we defaulting all empty envs to prod?

Reply:
we default to production right now, since that's what SDKs will send by default when no environment is specified.